src/renderer/components/TimelineSpace/Contents/Cards/Toot.vue

```diff
@@ -593,9 +597,14 @@ export default {
         }
       }
 
-      .reblogger-name {
+      .reblogger-name /deep/ {
         font-size: calc(var(--base-font-size) * 0.86);
         cursor: pointer;
+
+        .emojione {
```

Comment:
__[stylelint]__ <sub>posted by Sider</sub>

Expected selector ".toot .detail .reblogger .reblogger-name /deep/ .emojione" to come before selector ".toot .detail .toot-header .user .display-name /deep/ .emojione" (no-descending-specificity)